container-images/tcib/base/os/tempest/run_tempest.sh

```diff
@@ -95,6 +95,16 @@ CONCURRENCY="${CONCURRENCY:-}"
 TEMPESTCONF_ARGS=""
 TEMPEST_ARGS=""
 
+function catch_error_if_debug {
+    echo "File run_tempest.sh has run into an error!"
+    sleep infinity
+}
+
+# Catch errors when in debug mode
+if [ ${TEMPEST_DEBUG_MODE} ]; then
```

Comment:
do we need a default value? All the other variables here have one

Reply:
This needs to be updated:
```
if [[ ${TEMPEST_DEBUG_MODE} == true ]]; then
```